public/app/plugins/panel/table2/module.tsx

```diff
@@ -35,5 +35,7 @@ export const plugin = new PanelPlugin<Options>(TablePanel)
       id: 'showHeader',
       name: 'Show header',
       description: "To display table's header or not to display",
+      defaultValue: true,
     });
-  });
+  })
+  .useCommonFieldConfig([StandardFieldConfigProperties.Thresholds]);
```

Comment:
```suggestion
  });
```

Reply:
why would table only include thresholds?  I think it would be everything no?

Reply:
ahh -- i see... this is on the panel, not the field.  Why would you want threshold on the table, not a cell value?

Reply:
@ryantxu this is to be removed. It's only for an example here :)